client/src/hooks/use-auth.test.tsx

```
@@ -0,0 +1,303 @@
+
+vi.mock('@/hooks/use-toast', () => ({
+  useToast: () => ({
+    toast: vi.fn(),
+  }),
+}));
+
+// Mock window.location.href
+const originalLocation = window.location;
+beforeAll(() => {
+  // @ts-ignore
+  delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)